earthfile2llb/converter.go

```diff
@@ -643,6 +643,29 @@ func (c *Converter) internalRun(ctx context.Context, args []string, secretKeyVal
 			return fmt.Errorf("Secret definition %s not supported. Must start with +secrets/ or be an empty string", secretKeyValue)
 		}
 	}
+	// Secret Files.
+	for _, secretFileDef := range secretFiles {
+		secretPath, secretID, ok := variables.ParseEscapedKeyValue(secretFileDef)
```

Comment:
I *think* that we don't need a special implementation for this. In the previous implementation anything after the first `=` is part of the value. So `key=value=something` parses to key=`key` and value=`value=something`. Wouldn't that be enough?